client/src/containers/header/menu/index.tsx

```diff
@@ -12,6 +12,17 @@ const Menu: FC = () => {
   return (
     <nav className="min-w-[150px]">
       <ul className="overflow-hidden py-2">
+        <li>
+          <Link
```

Comment:
Do not use `Link` component in this case when is an external link. `Link` is used for internal routes as it is a wrap for an `a` that allows prefetching pages in the background. More info: https://nextjs.org/docs/app/api-reference/components/link

Reply:
Gotcha! Fix here 3f27e447d25b830cc4e84ab73577576e8558d918